extensions/libraries/redcore/database/sqlparser/sqltranslation.php

```diff
@@ -183,6 +183,58 @@ public static function buildTranslationQuery($sql = '', $prefix = '')
 		$db = JFactory::getDbo();
 
 		$selectedLanguage = !empty($db->forceLanguageTranslation) ? $db->forceLanguageTranslation : JFactory::getLanguage()->getTag();
+		$queryArray       = explode(',', (string) $sql);
+		$hashValues       = array();
+
+		// Replace long amount of numeric values if found. They do not have any affects for translation parser
+		if (count($queryArray) > 50)
```

Comment:
why only > 50?

Reply:
It can be less if you want.
In fact, we need to avoid an enormous amount of symbols in a query. For example, in my test, it was more than 200k letters. 
I think it's not so hard to parse a query with 50 numeric values. 
I'm replacing values to a hash key which usually has about 25 symbols, so make sense use it for much more longer examples.